Adapt README.md (#126)
echemdb/website: README.md

conda env update --name <your_env_name> --file environment.yml
```

Clone the repository and install echemdb

```
git clone https://github.com/echemdb/website.git
cd website
pip install -e .
```

# Python API

The current state of the website can be downloaded and stored in a database.

```python
>>> from echemdb.data.cv.database import Database
>>> db = Database()
```

> Filtering the database for entries having specific properties, i.e., containing Pt as working electrode material, returns a new database.

Reviewer: ```suggestion
Filtering the database for entries having specific properties, e.g., containing Pt as working electrode material, returns a new database.
```

```python
>>> db_filtered = db.filter(lambda entry: entry.electrochemical_system.electrodes.working_electrode.material == 'Pt')
```

A single entry can be retrieved with the identifiers provided on the website 
(see for example [engstfeld_2018_polycrystalline_17743_f4b_1](https://echemdb.github.io/website/cv/entries/engstfeld_2018_polycrystalline_17743_f4b_1/))

```python
>>> entry = db['engstfeld_2018_polycrystalline_17743_f4b_1']
```

Each entry has information about its source

```python
>>> entry.source # or entry['source']
{'citation key': 'engstfeld_2018_polycrystalline_17743', 'curve': 1, 'url': 'https://doi.org/10.1002/chem.201803418', 'figure': '4b', 'version': 1}
```

> Among other metadata, the entry also has information on the original figure properties (`entry.figure_description`) and the `entry.electrochemical_system` in general.

Reviewer: ```suggestion
Among other metadata, the entry also has information on the original publication's figure properties (`entry.figure_description`) and the `entry.electrochemical_system` in general.
```

> The data related to an entry can be returned as a [pandas](https://pandas.pydata.org/) dataframe (values are provided in SI units) and can be stored as CSV file (or any other format supported by pandas).

Reviewer: ```suggestion
The data related to an entry can be returned as a [pandas](https://pandas.pydata.org/) dataframe (values are provided in SI units) and can be stored as a CSV file (or any other format supported by pandas).
```

```python
>>> entry.df() 
           t	        E	       j
0	0.000000	-0.196962	0.043009
1	0.011368	-0.196393	0.051408
...

> # Custom or original figure axes units can be set specifically 

Reviewer: ```suggestion
# Custom or original figure axes' units can be requested explicitly 
```

> # entry.df(xunit='original', yunit='mA / m2')

Reviewer: Why is this commented out?

Author: I thought everything would be too cluttered. I included it now since it is an interesting aspect.

>>> entry.df().to_csv('../testtesttest.csv', index=False)
```

> The reference electrode to the potential axis `E` is provided in the `entry.data_description`

Reviewer: Not sure but should it be "for" the potential axis?

Author: I changed to "of the potential axis"

```python
>>> entry.data_description.axes.E.reference
'RHE'
```

The data can be visualized in a plotly figure, with preferred axis units (default is SI):

```python
>>> entry.plot(xunit='V', yunit='original')
```

> <!-- ![doc/images/readme_demo_plot.png](doc/images/readme_demo_plot.png)-->

Reviewer: ```suggestion
```
